modules/admin/ref-rbac-rest-api-endpoints.adoc

```diff
@@ -1,7 +1,209 @@
 [id='ref-rbac-rest-api-endpoints_{context}']
 = Supported RBAC REST API endpoints
 
-The RBAC REST API provides the following endpoints for managing permission policies and roles in the {product-short} and for retrieving information about the policies and roles.
+The RBAC REST API provides the following endpoints for managing roles, permissions, and conditional policies in the {product-short} and for retrieving information about the roles and policies.
+
+== Roles
+
+The RBAC REST API supports the following endpoints for managing roles in the {product}.
+
+[GET] /api/permission/roles::
```

Comment:
It took me a while to get used to this format and the sentence fragments, but it might actually be cleaner / easier to read this way. Disregard previous suggestion.